src/components/berths/SelectedBerth.js

```diff
@@ -45,15 +45,15 @@ const SelectedBerth = ({ berth, index, moveUp, moveDown, first, last, deselectBe
         </span>
 
         <DeselectButton type="button" onClick={() => deselectBerth(berth.identifier)}>
-          <Icon name="times" width="30" />
+          <Icon name="times" width="30px" />
```

Comment:
This however is fine because it's a prop for react component which is used in Icon as such:

svgStyle={{ width, height, fill }}